frontend/view-edit-client/interactions/referral-interaction-inputs.component.tsx

```diff
@@ -49,6 +50,11 @@ const ReferralInteractionInputs = React.forwardRef<
     props.setName(partnerService ? partnerService.name : "");
   }, [partnerService, props.setName]);
 
+  const getPartnerLabel = (partner) => {
+    const phoneLabel = partner.phone ? `; ${formatPhone(partner.phone)}` : "";
```

Comment:
I'm open to suggestions for another divider besides the `;`. I tried a few and this is the one I hated the least

Reply:
Looks fine in your screenshot